Collectors/DetailedCollector.py

```diff
@@ -382,9 +476,10 @@ def process(self, data, addr, port):
             stream_type = chr(decoded_gstream.ident >> 56)
             if stream_type == "T":
                 self.process_tcp(decoded_gstream, addr)
-            print(decoded_gstream)
-
-
+         
+         
+            # process the gstream
+            self.process_gstream(decoded_gstream, addr,sid)
```

Comment:
I think this needs to be surrounded by a:
```
elif stream_type == "C":
```

Reply:
sure!